.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 node_js:
   - '8'
-  - '10'
\ No newline at end of file
+  - '10'
+build:
+  - npm ci && npm run build
```

Comment:
Will this still run `npm install` before `npm ci` because we haven't overridden the `install` section?

```yml
install:
  - npm ci
build:
  - npm run build
```

Reply:
good question! i'll update that